src/PosthogAnalytics.ts

```diff
@@ -198,6 +205,9 @@ export class PosthogAnalytics {
         if (["layout", "useCompactLayout"].includes(settingsPayload.settingName)) {
             this.onLayoutUpdated();
         }
+        if (["feature_rust_crypto"].includes(settingsPayload.settingName)) {
+            this.updateCryptoSuperProperty();
+        }
```

Comment:
This code will never fire, no? Given the feature is not runtime mutable and must be set via config.

Reply:
Yes I simplified all, it will probably not be fully live ever. Just using the matrixClient passed on load/login/account data and read the crypto version